components/ProcessingSplash.qml

```diff
@@ -29,21 +29,23 @@
 import QtQuick 2.9
 import QtQuick.Window 2.1
 import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
 import QtQuick.Layouts 1.1
 
 import "../components" as MoneroComponents
 
 Rectangle {
     id: root
-    color: MoneroComponents.Style.blackTheme ? "white" : "transparent"
+    color: MoneroComponents.Style.blackTheme ? "black" : "white"
     visible: false
+    radius: 10
```

Comment:
Looks inconsistent with the other dialogs

Reply:
Plan is to update the other dialogs too over time:

https://www.figma.com/file/DplJ2DDQfIKiuRvolHX2hN/Monero-GUI?node-id=2%3A4925

Reply:
Ok, let's use radius.